diff -uNr moloch-master/single-host/etc/config.ini.template moloch-master-new/single-host/etc/config.ini.template
--- moloch-master/single-host/etc/config.ini.template	2014-05-01 08:24:10.000000000 -0500
+++ moloch-master-new/single-host/etc/config.ini.template	2014-05-06 10:17:02.175163348 -0500
@@ -96,7 +96,7 @@
 # delete the session records in the database. It is recommended this value is 
 # between 5% and 10% of the disk.  Database deletes are done by the 
 # daily.sh script
-freeSpaceG = 600
+freeSpaceG = 5 
 
 # The port to listen on, by default 8005
 viewPort = 8005
@@ -246,7 +246,7 @@
 # Classes of nodes
 # Can override most default values, and create a tag call node:<classname>
 [test]
-freeSpaceG = 200
+freeSpaceG = 5 
 
 ##############################################################################
 # Nodes
