--- moloch-master/easybutton-build.sh	2014-05-01 08:24:10.000000000 -0500
+++ moloch-master-new/easybutton-build.sh	2014-05-02 15:16:32.216279430 -0500
@@ -101,7 +101,7 @@
 # Not sure why this is required on some platforms
 if [ ! -f "GeoIP-$GEOIP/libGeoIP/.libs/libGeoIP.a" ]; then
   if [ -f "/usr/bin/libtoolize" ]; then
-    (cd GeoIP-$GEOIP ; libtoolize -f)
+    (cd GeoIP-$GEOIP ; libtoolize -f && ./bootstrap)
   fi
   (cd GeoIP-$GEOIP ; ./configure --enable-static; make)
 else
